src/registrar/management/commands/transfer_transition_domains_to_domains.py

```diff
@@ -0,0 +1,229 @@
+    > Found existing domain entry for: {transition_domain_name}, {current_state}
+    {termColors.ENDC}"""
+                    )
+                if transition_domain_status != current_state:
+                    if (
+                        transition_domain_status
+                        == TransitionDomain.StatusChoices.ON_HOLD
+                    ):
+                        existingEntry.place_client_hold(ignoreEPP=True)
+                    else:
+                        existingEntry.revert_client_hold(ignoreEPP=True)
+                    existingEntry.save()
+                    updated_domain_entries.append(existingEntry)
+                    # DEBUG:
+                    if debug_on:
+                        logger.info(
+                            f"""{termColors.WARNING}
+    >> Updated {transition_domain_name} state from
+    '{current_state}' to '{existingEntry.state}{termColors.ENDC}'"""
+                        )
+            except Domain.DoesNotExist:
+                # no matching entry, make one
+                newEntry = Domain(
+                    name=transition_domain_name, state=transition_domain_status
+                )
+                to_create.append(newEntry)
+
+                # DEBUG:
+                if debug_on:
+                    logger.info(
+                        f"{termColors.OKCYAN} Adding entry: {newEntry} {termColors.ENDC}"  # noqa
+                    )
+            except Domain.MultipleObjectsReturned:
+                logger.info(
+                    f"""
+{termColors.FAIL}
+!!! ERROR: duplicate entries exist in the
+Domain table for domain:
+{transition_domain_name}
+----------TERMINATING----------"""
+                )
+                import sys
+
+                sys.exit()
+            except TransitionNotAllowed as err:
+                skipped_domain_entries.append(transition_domain_name)
+                logger.info(
+                    f"""{termColors.FAIL}
+Unable to change state for {transition_domain_name}
+    TRANSITION NOT ALLOWED error message (internal):
+    {err}
+    ----------SKIPPING----------"""
+                )
+
+            # DEBUG:
+            if debug_on or debug_max_entries_to_parse > 0:
+                if (
+                    total_rows_parsed > debug_max_entries_to_parse
+                    and debug_max_entries_to_parse != 0
+                ):
+                    logger.info(
+                        f"""{termColors.WARNING}
+                        ----PARSE LIMIT REACHED.  HALTING PARSER.----
+                        {termColors.ENDC}
+                        """
+                    )
+                    break
+
+        Domain.objects.bulk_create(to_create)
+
+        total_new_entries = len(to_create)
+        total_updated_domain_entries = len(updated_domain_entries)
+
+        logger.info(
+            f"""{termColors.OKGREEN}
+
+============= FINISHED ===============
+Created {total_new_entries} transition domain entries,
```

Comment:
can we fix this indenting? and the other ones below that do the same thing

Reply:
Indents fixed